tests/attested_tls/sgxlkl_enclave/oe_public/bits/result.h

```diff
@@ -0,0 +1,375 @@
+
```

Comment:
Why do we need this file?

Reply:
to define _oe_result I think, do you think we need to remove it and define by ourself?

Reply:
removed